public/components/layout.js

```diff
@@ -136,19 +160,38 @@ class UserResource extends HTMLElement {
                 return obj
             })
             .catch(err => {
-                alert("Target URI could not be resolved."
-                    + " Certain data previews will not be available."
-                    + " Applications that will use the data you are about to create"
-                    + " will note be able to gather additional information about this targeted resource."
-                    + " You can supply a different URI or continue with this one.")
+                document.getElementById("notResolvedMessage").classList.toggle("show")
                 uriPreview.innerHTML = `<pre>{Not Resolvable}</pre>`
                 localStorage.setItem('userResource', objCreator.value? JSON.stringify({'@id':target, 'creator':objCreator.value}): JSON.stringify({'@id':target}))
                 return null
             })
+        this.validateContext(targetObj)
         //This might help mobile views
         //window.scrollTo(0, confirmURI.offsetTop)
     }
 
+    validateContext(target_object){
```

Comment:
This should only fire for objects supplied for navPlace (from navPlace.html). We do not need to validate the context of the supplied resource when creating an Annotation (from annotate.html).